src/bp-templates/bp-nouveau/buddypress/groups/groups-loop.php

```diff
@@ -6,91 +6,165 @@
+$meta_privacy       = ! bb_platform_group_element_enable( 'group-privacy' ) ? 'meta-privacy-hidden' : '';
+$meta_group_type    = ! bb_platform_group_element_enable( 'group-type' ) ? 'meta-group-type-hidden' : '';
+$group_members      = ! bb_platform_group_element_enable( 'members' ) ? 'group-members-hidden' : '';
+$join_button        = ! bb_platform_group_element_enable( 'join-buttons' ) ? 'group-join-button-hidden' : '';
+$group_alignment    = bb_platform_group_grid_style( 'left' );
+$group_cover_height = function_exists( 'bb_get_group_cover_image_height' ) ? bb_get_group_cover_image_height() : 'small';
+
+$group_cover_height_class = 'cover-large';
+if ( 'small' === $group_cover_height ) {
+	$group_cover_height_class = '';
+}
+?>
+
 <?php if ( bp_has_groups( bp_ajax_querystring( 'groups' ) ) ) : ?>
 
 	<?php bp_nouveau_pagination( 'top' ); ?>
 
-	<ul id="groups-list" class="<?php bp_nouveau_loop_classes(); ?>">
-
-	<?php
-	while ( bp_groups() ) :
-		bp_the_group();
-		?>
-
-		<li <?php bp_group_class( array( 'item-entry' ) ); ?> data-bp-item-id="<?php bp_group_id(); ?>" data-bp-item-component="groups">
-			<div class="list-wrap group-list">
-
-				<div class="group-list-header">
-					<?php if ( ! bp_disable_group_cover_image_uploads() ) { ?>
-						<?php
-						$group_cover_image_url = bp_attachments_get_attachment(
-							'url',
-							array(
-								'object_dir' => 'groups',
-								'item_id'    => bp_get_group_id(),
-							)
-						);
-						$has_default_cover     = bb_attachment_get_cover_image_class( bp_get_group_id(), 'group' );
-						?>
-						<div class="bs-group-cover only-grid-view <?php echo esc_attr( $has_default_cover ); ?>"><a href="<?php bp_group_permalink(); ?>"><img src="<?php echo esc_url( $group_cover_image_url ); ?>"></a></div>
-					<?php } ?>
+	<ul id="groups-list" class="<?php bp_nouveau_loop_classes(); ?> <?php echo esc_attr( $cover_class . ' ' . $group_alignment ); ?> groups-dir-list">
+
+		<?php
+		while ( bp_groups() ) :
+			bp_the_group();
+			?>
+
+			<li <?php bp_group_class( array( 'item-entry' ) ); ?> data-bp-item-id="<?php bp_group_id(); ?>" data-bp-item-component="groups">
+				<div class="list-wrap">
+
+					<?php
+					if ( ! bp_disable_group_cover_image_uploads() ) {
+						if ( bb_platform_group_element_enable( 'cover-images' ) ) {
+							$group_cover_image_url = bp_attachments_get_attachment(
+								'url',
+								array(
+									'object_dir' => 'groups',
+									'item_id'    => bp_get_group_id(),
+								)
+							);
+							$has_default_cover     = function_exists( 'bb_attachment_get_cover_image_class' ) ? bb_attachment_get_cover_image_class( bp_get_group_id(), 'group' ) : '';
+							?>
+							<div class="bs-group-cover only-grid-view <?php echo esc_attr( $has_default_cover . ' ' . $group_cover_height_class ); ?>"><a href="<?php bp_group_permalink(); ?>"><img src="<?php echo esc_url( $group_cover_image_url ); ?>"></a></div>
```

Comment:
follow this here: https://github.com/buddyboss/buddyboss-platform/pull/3339/files#r817572973